src/openrct2-ui/windows/Research.cpp

```diff
@@ -402,11 +420,31 @@ void window_research_development_page_paint(rct_window* w, rct_drawpixelinfo* dp
     rct_string_id lastDevelopmentFormat;
```

Comment:
Move this to be inside the if, so we reduce its scope, as it's not needed anymore later. Also either initialize it to `STR_RESEARCH_SCENERY_LABEL` and change if `type != RESEARCH_ENTRY_TYPE_SCENERY` or to something invalid, so we know in case we break this someday.